acasclient/acasclient.py

```diff
@@ -440,6 +440,25 @@ def add_file_to_lot(self, lot_corp_name, file, file_type, writeup=None):
         """
         return self.add_files_to_lot(lot_corp_name, [{"file": file, "file_type": file_type, "writeup": writeup}])
 
+    def _get_cmpdreg_files_content(self, files):
+        """Get the content of cmpdreg files
+
+        Pass an array of cmpdreg_files to download and fill their content attribute
+
+        Args:
+            files: An array of cmpdreg_files to download in the format
+                "subdir": The subdir of the file (usually lot_corp_name)
```

Comment:
Can we mention that this is optional? Or what do you think about totally removing it -  In the request we only pass the URL of the file, and we are returning the entire `files` object as it is with the "content" parameter added.